src/meta/src/rpc/cloud_provider.rs

```diff
@@ -47,12 +49,15 @@ impl AwsEc2Client {
     }
 
     /// `service_name`: The name of the endpoint service we want to access
+    /// `az_ids`: The AZs in which the service is available (deprecated)
     pub async fn create_aws_private_link(
```

Comment:
Since `PrivateLink` already has a field name `provider`, how about rename it to `make_private_link` and return an error if the `provider` is not `aws` to make it more general.

Reply:
This is a method in `AwsEc2Client`, so we may not make it general. I add a `match` statement  in `create_connection` of `DdlService` to match `provider` then return error if it is not `aws`.